### PR TITLE
Adds new case studies - SMG and IC #328

### PR DESCRIPTION
**Pull request by Cleop**
Creates new SMG and IC portfolio pages #328 including links to them from the main portfolio page.

### CODE REVIEW

File: redesign/cs-emf.html [hunk 1]
```diff
@@ -107,9 +107,9 @@ <h3 class="b f5">Project Length</h3>
         </p>
         <h3 class="b f5">Implementation Team</h3>
         <p class="f6">
-          Scrum master
-          System Architect & Q&A
-          Mid-level Developer
+          Scrum master<br>
+          System Architect &amp; QA<br>
+          Mid-level Developer<br>
```
Comment (iteles) [inline redesign/cs-emf.html]: All of the case study content needs reviewing and this is with me.

In  the meantime, I don't think it makes any  sense to include the 'level' of developer for the website, '2  Developers' should suffice.  (also this 'level' is also incorrect - as I say will have to review all the content).

@Cleop Thinking we just merge this in and I review all the case studies at the end seeing as we don't have permission to publish these on our website yet anyway.

File: redesign/cs-smg.html [hunk 2]
```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
```
Comment (iteles) [inline redesign/cs-smg.html]: We need to rethink these routes. Whilst we don't have a back-end, the html file names will be used as routes and these need to be explanatory, not cryptic for SEO purposes.

e.g. `.com/case-studies/science-museum-online collections`

File: redesign/cs-smg.html [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+    <section class="pt5 pb3 pb3-ns ph6-ns center tc shadow-3">
+      <img src="./img/common/science-museum-logo.jpg" alt="Science Museum Group logo" class="w4 pt4 mt4"/>
+      <h2 class="dwyl-light-gray f4">The Science Museum Group (SMG)</h2>
+      <h1 class="dwyl-dark-gray f3-plus w-90 tc center">Online Databank for Museum Collections</h1>
+    </section>
+
+    <section class="pt2 pb3 pv4-ns w-80 center">
+      <div class="w-40-ns dtc-ns v-top-ns">
+        <img src="./img/common/science-museum-screenshot1.png" alt="A screenshot of the project search page" class="w-60-ns pl5-ns">
```
Comment (iteles) [inline redesign/cs-smg.html]: This actual screenshot is incomplete - it's very clearly cut on the right  hand side (can''t comment on the image file below, so have to add here, sorry).

File: redesign/cs-smg.html [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+        <img src="./img/common/science-museum-screenshot2.png" alt="A presentation of the project on stage" class="w-60-ns pt4 pl5-ns dn db-ns">
```
Comment (iteles) [inline redesign/cs-smg.html]: All `alt` text is great, but needs to be more explanatory and include key words like 'science museum' in each one - think SEO.

File: redesign/cs-smg.html (hunk 3 above)
Comment (Cleop) [reply redesign/cs-smg.html]: @markwilliamfirth  please can you provide another screenshot on #328?

File: redesign/cs-smg.html (hunk 2 above)
Comment (Cleop) [reply redesign/cs-smg.html]: Proposing: /case-studies/ellen-macarthur-foundation, /case-studies/inclusive-classrooms and /case-studies/science-museum-online for now?

Comment (markwilliamfirth) [reply redesign/cs-smg.html]: We should probably have a convention:

case-studies/client-name/project-name

e.g.
/case-studies/ellen-macarthur-foundation/ce100
/case-studies/inclusive-classrooms/[project-name]
case-studies/science-museum-group/online-collections

can't remember what exact project names are, but replace the end with the project name on the copy

Comment (markwilliamfirth) [reply redesign/cs-smg.html]: forgot to tag @Cleop

File: redesign/cs-smg.html (hunk 3 above)
Comment (markwilliamfirth) [reply redesign/cs-smg.html]: I will open another issue for this as it isn't a blocker

File: redesign/cs-emf.html (hunk 1 above)
Comment (markwilliamfirth) [reply redesign/cs-emf.html]: I think having the level of developer is useful for a prospective client as it gives them an idea of how much time it took what level of team to create the product

**Comment by finnhodgkin** [comment]
Conflicts resolved @iteles :+1:

**Comment by markwilliamfirth** [comment]
Routes have been updated and conflicts resolved so merging this in

**Pull request merged**